registry/txt_test.go

```diff
@@ -703,8 +703,10 @@ func newEndpointWithOwner(dnsName, target, recordType, ownerID string) *endpoint
 func newEndpointWithOwnerAndLabels(dnsName, target, recordType, ownerID string, labels endpoint.Labels) *endpoint.Endpoint {
 	e := endpoint.NewEndpoint(dnsName, recordType, target)
 	e.Labels[endpoint.OwnerLabelKey] = ownerID
-	for k, v := range labels {
-		e.Labels[k] = v
+	if labels != nil { //nolint:gosimple
+		for k, v := range labels {
+			e.Labels[k] = v
+		}
```

Comment:
```suggestion
	for k, v := range labels {
		e.Labels[k] = v
```
Is this really needed?
https://play.golang.org/p/G1cwJb33Dvl